docs/analytical_engine/vertex_centric_models.md

```diff
@@ -1,20 +1,59 @@
-# Vertex Centric Models
+# Vertex-Centric Model
 
-biref intro to vertex centric models, history;
+In a single machine environment, developers can easily implement graph analytics algorithms as they have a global view of the graph and can freely iterate through all vertices and edges. When the size of graph data grows beyond the memory capacity of a single machine, graph data must be partitioned to distributed memory, leading to the indivisibility of the whole graph structure. 
 
-## GAS Model
+To allow developers to succinctly express graph analytics algorithms under such environment, the *vertex-centric programming model* have been developed. Specifically, a graph analytics algorithm iteratively executes a user-defined program over vertices of a graph. The user-defined vertex program
+typically takes data from other vertices as input, and the
+resultant output of a vertex is sent to other vertices. Vertex programs are
+executed iteratively for a certain number of rounds, or until a convergence condition is
+satisfied. As opposed to *global* perspective of the graph, vertex-centric models
+employ a local, vertex-oriented perspective.
 
-intro and figure
+The philosophy of vertex-centric model encourages many programming models, including the [Pregel model](https://research.google/pubs/pub37252/) proposed by Google and the [GAS model](https://www.usenix.org/conference/osdi12/technical-sessions/presentation/gonzalez). These programming models
+have widely applied in various graph processing systems, such as Giraph, GraphX and PowerGraph.
```

Comment:
As a history, you may want to give a brief intro to GAS, just like Pregel. 
for GAS, it is OK not to give code.